packages/drawer/src/views/Drawer.tsx

```diff
@@ -489,7 +495,9 @@ export default class DrawerView extends React.Component<Props> {
     // Check if the drawer width is available to avoid division by zero
     eq(this.drawerWidth, 0),
     0,
-    abs(divide(this.translateX, this.drawerWidth))
+    Platform.OS === 'windows' || Platform.OS === 'macos'
```

Comment:
Maybe you could add a comment about why this is needed

Reply:
Moved the platform check to Overlay.tsx and added some comments.